src/components/expression/expression-loader.js

```diff
@@ -170,6 +169,22 @@ genotet.ExpressionLoader.prototype.loadTfaProfile_ = function(fileName,
 };
 
 /**
+ * Loads expression data list into genotet.data.expressionFiles.
```

Comment:
Sorry I don't understand your question. I mean if you are calling a function that loads data to genotet.data, then shouldn't this function belong to genotet.data?